src/index.ts

```diff
@@ -33,9 +33,9 @@ class ExoplatformBot {
    */
   request(
     path: string,
-    body: object = {},
-    method: string = 'GET',
-    moreOptions?: object
+    body: object | null = null,
+    method: string = body ? 'POST' : 'GET',
```

Comment:
This is wrong as eXo Platform's API is REST. You can send a body with other HTTP methods, like PUT.

Reply:
It's just the *default* value :smile: The idea is just to remove the necessity of the `'POST'` here `this.request(yourUrl, yourBody, 'POST')`, but it's always possible to do so ^^